lib/gds_api/email_alert_api.rb

```diff
@@ -127,22 +127,22 @@ def get_subscription(id)
 
   # Get Subscriptions for a Subscriber
   # #
-  # @param string Subscriber address
+  # @param string Subscriber id
```

Comment:
This should be updated from string (and there's another instance below)

Reply:
Ah yes, fixed